transactions/admin/fake_admin/create_set.cdc

```diff
@@ -0,0 +1,35 @@
+import TopShot from 0xFAKETOPSHOTADDRESS
```

Comment:
it is my understanding that the only thing different from the tx and the original create set tx is the Topshot address. If this is the case we shouldn't need to write a new tx file. Yo just need to replace the address with the fake address

Reply:
same applies for other txs

Reply:
Good point, I have removed the fake topshot address and instead I just use a modified env for the tests